modules/core/arrow-test/src/main/kotlin/arrow/test/laws/OrderLaws.kt

```diff
@@ -4,11 +4,16 @@ import arrow.typeclasses.Eq
 import arrow.typeclasses.Order
 import io.kotlintest.properties.Gen
 import io.kotlintest.properties.forAll
+import java.util.*
 
 object OrderLaws {
 
+  val random = Random()
```

Comment:
This retained random never gets reset, so it's only seeded once when `OrderLaws ` is first created. We want entropy for every run, so it needs to be created inside each one instead.